src/main/java/net/jextra/fauxjo/Table.java

```diff
@@ -164,6 +164,28 @@ public String buildBasicSelectStatement( String clause )
         return String.format( "select %s from %s %s", fields, fullTableName, trimmedClause );
     }
 
+    public String buildBasicSelectStatement( String alias, String clause )
+    {
+        String trimmedClause = "";
+        if ( clause != null && !clause.trim().isEmpty() )
+        {
+            trimmedClause = clause;
+        }
+        String fields = "*";
+        try
+        {
+            Map<String, FieldDef> beanFieldDefs = BeanDefCache.getFieldDefs( beanClass );
+            String prefix = alias != null && !alias.isEmpty() ? alias + "." : "";
+            fields = prefix + String.join( ", " + prefix, beanFieldDefs.keySet() );
+            return String.format( "select %s from %s %s %s", fields, fullTableName, alias, trimmedClause );
+        }
+        catch ( SQLException sqle )
+        {
+            // Do nothing, default to *
```

Comment:
I understand that the other method is not logging the error, but If it were me I would probably log the error at warn level so that the user knows that it failed and defaulted to *.

Reply:
Seems reasonable, I'll add that in.